worker agents will fail to start with v1.19.x k3s

<!-- Thanks for helping us to improve k3OS! We welcome all bug reports. Please fill out each area of the template so we can better help you.  ***You can delete this message portion of the bug report.*** -->

**Version (k3OS / kernel)**
<!-- Provide output from `k3os --version` OR `sh -c '. /etc/os-release;echo $VERSION_ID'` -->
<!-- Provide output from `uname --kernel-release --kernel-version` -->
k3os version v0.19.2-dev.1
5.4.0-48-generic

**Architecture**
<!-- Provide output `uname --machine ` -->
x86_64

**Describe the bug**
<!-- A clear and concise description of what the bug is. -->
Worked failed to fully upgrade when running the `k3s-install.sh` for the `k3s-service`.

**To Reproduce**

Attempt to upgrade from v0.11.1 to v0.19.2-dev.1, worker k3s will fail to start.
```
nuc-2 [~]$ sudo k3os --debug config
DEBU[0000] running cmd `rc-service ntpd start`          
 * WARNING: ntpd has already been started
DEBU[0000] running cmd `sleep 5`                        
DEBU[0005] Running /usr/libexec/k3os/k3s-install.sh [/usr/libexec/k3os/k3s-install.sh agent --node-label k3os.io/mode=local --node-label k3os.io/version=v0.19.2-dev.1] [INSTALL_K3S_NAME=service INSTALL_K3S_SKIP_DOWNLOAD=true INSTALL_K3S_BIN_DIR=/sbin INSTALL_K3S_BIN_DIR_READ_ONLY=true K3S_URL="https://nuc-1:6443"
 K3S_CLUSTER_SECRET="demo"
] 
[ERROR]  Only https:// URLs are supported for K3S_URL (have "https://nuc-1:6443"
)
ERRO[0005] exit status 1                                
```

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->
All agents start after being upgraded+rebooted.

**Actual behavior**
<!-- A clear and concise description of what actually happened. -->
Worker agent failed to restart after being upgraded then rebooted.

**Additional context**
<!-- Add any other context about the problem here, such as relevant hardware or virtualization detail -->
Recent changes in `$K3S_URL` validation in the install script have broken this weirdness:
- https://github.com/rancher/k3os/blob/v0.19.2-dev.1/pkg/cc/funcs.go#L125
- https://github.com/rancher/k3os/blob/v0.19.2-dev.1/pkg/cc/funcs.go#L132
- https://github.com/rancher/k3os/blob/v0.19.2-dev.1/pkg/cc/funcs.go#L134

--- a/pkg/cc/funcs.go
+++ b/pkg/cc/funcs.go
@@ -122,16 +122,16 @@ func ApplyK3S(cfg *config.CloudConfig, restart, install bool) error {
 			args = append(args, "server")
 		}
 	} else {
-		vars = append(vars, fmt.Sprintf("K3S_URL=\"%s\"\n", cfg.K3OS.ServerURL))
+		vars = append(vars, fmt.Sprintf("K3S_URL=%s", cfg.K3OS.ServerURL))
 		if len(args) == 0 {
 			args = append(args, "agent")
 		}
 	}
 
 	if strings.HasPrefix(cfg.K3OS.Token, "K10") {
-		vars = append(vars, fmt.Sprintf("K3S_TOKEN=\"%s\"\n", cfg.K3OS.Token))
+		vars = append(vars, fmt.Sprintf("K3S_TOKEN=%s", cfg.K3OS.Token))
 	} else if cfg.K3OS.Token != "" {
-		vars = append(vars, fmt.Sprintf("K3S_CLUSTER_SECRET=\"%s\"\n", cfg.K3OS.Token))
+		vars = append(vars, fmt.Sprintf("K3S_CLUSTER_SECRET=%s", cfg.K3OS.Token))
 	}
 
 	var labels []string

